spec/lib/court_data_adaptor/name_parser_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'court_data_adaptor'
+
+RSpec.describe CourtDataAdaptor::NameParser do
+  describe '#parts' do
+    subject(:parts) { described_class.new(term).parts }
+
+    let(:term) { 'whatever' }
+
+    it 'is a hash' do
+      is_expected.to be_a Hash
+    end
+
+    context 'when one term' do
+      let(:term) { 'fred   ' }
+
+      it 'has only first name' do
+        is_expected.to eql(first: 'fred', last: nil, middle: nil)
+      end
+    end
+
+    context 'when two terms' do
+      let(:term) { 'fred   alberts   ' }
```

Comment:
Is there a specific reason why multiple spaces are being used?

Reply:
yep, testing whitespace removal